generator/templates/stories/{{ folderName }}.mdx.hbs

```diff
@@ -1,18 +1,31 @@
-import { ArgTypes, Meta, Description, Primary, Stories, Title } from '@storybook/blocks';
+import {
+  ArgTypes,
+  Canvas,
+  Meta,
+  Description,
+  Title,
+  Subtitle,
+} from "@storybook/blocks";
+import { ComponentDetails, TaggedReleases } from "@spectrum-css/preview/blocks";
 
 import * as {{ pascalCase name }}Stories from './{{ folderName }}.stories';
 
-<Meta of={{{ pascalCase name }}Stories} title="Docs" />
+<Meta of={ {{ pascalCase name }}Stories } title="Docs" />
 
-<Title of={{{ pascalCase name }}Stories} />
-<Description of={{{ pascalCase name }}Stories} />
+<Title of={ {{ pascalCase name }}Stories } />
+<Subtitle of={ {{ pascalCase name }}Stories } />
+<ComponentDetails />
 
-<Primary of={{{ pascalCase name }}Stories} />
+<Description of={ {{ pascalCase name }}Stories } />
+
+<Stories of={ {{ pascalCase name }}Stories } />
```

Comment:
Could you explain the changes here? (it can literally be here in the PR or even in Slack) I don't have a good grasp on what this hbs file is doing (this is the first time I've seen it!)

Reply:
Great question, we haven't had to use the generator in a while because we've been focusing on updates and not new components. We can run `yarn new` to generate a templated component as a starting point. I added the spaces because the tooling can't interpolate the variable syntax `{{` with the `{` in front of it.